orchagent/sainet.h

```diff
@@ -0,0 +1,105 @@
+// Header file definine base types and simple helper functions for network in SAI
+// Should keep the dependency as minimal as possible
+//
+#pragma once
+
+#include <assert.h>
+#include <memory.h>
+#include <net/if.h>
+extern "C" {
+#include "sai.h"
+}
+#include "ipaddress.h"
+#include "ipprefix.h"
+
+inline static sai_ip_address_t& copy(sai_ip_address_t& dst, const swss::ip_addr_t& src)
+{
+    switch(src.family)
+    {
+        case AF_INET:
+            dst.addr_family = SAI_IP_ADDR_FAMILY_IPV4;
+            dst.addr.ip4 = src.ip_addr.ipv4_addr;
+            break;
+        case AF_INET6:
+            dst.addr_family = SAI_IP_ADDR_FAMILY_IPV6;
+            memcpy(dst.addr.ip6, src.ip_addr.ipv6_addr, 16);
+            break;
+        default:
+            assert(false); // unreachable code
```

Comment:
asserts in release code "do nothing"
please replace with SWSS_LOG_ERROR("..") with explanation and then throw or exit(-1)